packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -103,12 +105,24 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
     setOpen(false);
   }, []);
 
+  const hidePanel = useCallback(() => {
+    setOpen(false);
+  }, []);
+
+  const ref = useOnClickOutside<HTMLDivElement>(hidePanel);
```

Comment:
Thanks for the review @brandones 
There is something I tried out here; 

```
const ref = useOnClickOutside<HTMLDivElement>( () => {
  console.log(‘clicked outside’);
}, open);
```

the surprising thing was I would see the log message each time I click inside the **search-input** or the `search-component` instead of seeing the log each time I click **outside** the component.

it was like the logic in `useOnClickOutside` was working inversely.  
I am also going to implement the above suggestions and update the PR 

thank you so much @brandones